### PR TITLE
Jewell tutorial 1

### PR DESCRIPTION
**Pull request by jbartonthomas**
I have started work on a tutorial describing Anaconda + Jupyter setup with an applied example showing how to call MALLET from python. I have not finished this, but the local jekyll build doesn't get all of the image pathing right and I think the github build will, so I'd like to get a build.

### CODE REVIEW

File: _config.yml [hunk 1]
```diff
@@ -49,3 +49,5 @@ defaults:
       category: posts
 
 # For local work: bundle exec jekyll serve --watch --baseurl '/proghistdev' or wherever else your local site is located
+
+theme: jekyll-theme-slate
```
Comment (jerielizabeth) [inline _config.yml]: So, one thing with the jekyll theme -- it appears that you are adding an alternative theme to the config file. The lesson will need to work with the default theming.

File: .gitignore [hunk 2]
```diff
@@ -1,2 +1,4 @@
 _site/
 .DS_Store
+.jekyll-metadata
```
Comment (jerielizabeth) [inline .gitignore]: I would also leave these out of the file pushed up to the repo.

**Review by jerielizabeth — COMMENTED** [review]
On additional question: do you need the notebook file for the lesson?

I wanted to share the example notebook with the lesson. Is assets the right place for that? I can just move it to a separate repository.

File: _config.yml (hunk 1 above)
Comment (jbartonthomas) [reply _config.yml]: Hmm. Now that I'm looking, I'm not sure where that line came from.

File: .gitignore (hunk 2 above)
Comment (jbartonthomas) [reply .gitignore]: Sounds good, I figured everyone would want gitignore these random build cache files, but I'll just go back to default.

**Comment by jerielizabeth** [comment]
"assets" would be a good place for the example notebook, yes! Sorry -- I may have missed the directory information.

**Comment by jerielizabeth** [comment]
It looks like you picked up a couple extra files in this pull request (lessons/exploratory-data-analysis-with-nlp.md and lessons/fetch-and-parse-data-with-openrefine.md). I'm not quite sure what's going on there, but you would be willing to pull those out of your commits?

**Pull request merged**